src/main/drivers/pwm_mapping.c

```diff
@@ -77,35 +77,35 @@ enum {
 
 #if defined(NAZE) || defined(OLIMEXINO) || defined(NAZE32PRO) || defined(STM32F3DISCOVERY) || defined(EUSTM32F103RC) || defined(PORT103R)
 static const uint16_t multiPPM[] = {
-    PWM1  | (MAP_TO_PPM_INPUT << 8),     // PPM input
-    PWM9  | (MAP_TO_MOTOR_OUTPUT << 8),      // Swap to servo if needed
-    PWM10 | (MAP_TO_MOTOR_OUTPUT << 8),     // Swap to servo if needed
-    PWM11 | (MAP_TO_MOTOR_OUTPUT << 8),
-    PWM12 | (MAP_TO_MOTOR_OUTPUT << 8),
-    PWM13 | (MAP_TO_MOTOR_OUTPUT << 8),
-    PWM14 | (MAP_TO_MOTOR_OUTPUT << 8),
-    PWM5  | (MAP_TO_MOTOR_OUTPUT << 8),      // Swap to servo if needed
-    PWM6  | (MAP_TO_MOTOR_OUTPUT << 8),      // Swap to servo if needed
-    PWM7  | (MAP_TO_MOTOR_OUTPUT << 8),      // Swap to servo if needed
-    PWM8  | (MAP_TO_MOTOR_OUTPUT << 8),      // Swap to servo if needed
+    PWM1  | (MAP_TO_PPM_INPUT << 8),     // PPM input - naze TIM2
+    PWM9  | (MAP_TO_MOTOR_OUTPUT << 8),  // Swap to servo if needed - naze TIM1
+    PWM10 | (MAP_TO_MOTOR_OUTPUT << 8),  // Swap to servo if needed - naze TIM1
+    PWM11 | (MAP_TO_MOTOR_OUTPUT << 8),  // naze TIM4
+    PWM12 | (MAP_TO_MOTOR_OUTPUT << 8),  // naze TIM4
+    PWM13 | (MAP_TO_MOTOR_OUTPUT << 8),  // naze TIM4
+    PWM14 | (MAP_TO_MOTOR_OUTPUT << 8),  // naze TIM4
+    PWM5  | (MAP_TO_MOTOR_OUTPUT << 8),  // Swap to servo if needed - naze TIM3
+    PWM6  | (MAP_TO_MOTOR_OUTPUT << 8),  // Swap to servo if needed - naze TIM3
+    PWM7  | (MAP_TO_MOTOR_OUTPUT << 8),  // Swap to servo if needed - naze TIM3
+    PWM8  | (MAP_TO_MOTOR_OUTPUT << 8),  // Swap to servo if needed - naze TIM3
```

Comment:
although the information regarding timers is useful to developers it will go stale over time - comments cannot be refactored and should be avoided where possible.